flask_annex/s3.py

```diff
@@ -40,11 +40,15 @@ def delete(self, key):
         self._client.delete_object(Bucket=self._bucket_name, Key=key)
 
     def delete_many(self, keys):
+        objects = tuple({'Key': key} for key in keys)
```

Comment:
probably worth noting that you're doing this first because `keys` may be an iterable